client/src/components/LoginForm.jsx

```diff
@@ -77,16 +79,19 @@ export default function LoginForm() {
                                             onClick={handleClickShowPassword}
                                             onMouseDown={handleMouseDownPassword}
                                             edge="end">
-                                            {values.showPassword ? <VisibilityOff /> : <Visibility />}
+                                            {showPassword ? <VisibilityOff /> : <Visibility />}
                                         </IconButton>
                                     </InputAdornment>
                                 } />
                         </FormControl>
                     </form>
-                    <Button variant="contained">Login</Button>
+                    <Button 
+                        disabled={!email.length || !password.length}
+                        onClick={(email.length && password.length) ? () => AuthUser.login(email, password) : null}
```

Comment:
Could you add this inside the `form` tag and have the event be triggered by `onSubmit` instead of `onClick`? That will allow users to submit the form by hitting enter and a bunch of other accessibility stuff.

Reply:
Done.